app/app_xdg.go

```diff
@@ -20,8 +20,10 @@ import (
 	"fyne.io/fyne/v2/theme"
 )
 
+var currentVariant = theme.VariantDark
+
 func defaultVariant() fyne.ThemeVariant {
-	return findFreedestktopColorScheme()
+	return currentVariant
```

Comment:
Hmm. I think this might be introducing a race condition given that it is modified in a different goroutine? Might be worth using something like an `atomic.Pointer` or a lock?